packages/features/bookings/Booker/Booker.tsx

```diff
@@ -124,24 +126,21 @@ const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: Book
       )}
       <div className="flex h-full w-full flex-col items-center">
         <div
-          style={resizeAnimationConfig[layout]?.[bookerState] || resizeAnimationConfig[layout].default}
+          ref={animationScope}
           className={classNames(
             // Size settings are abstracted on their own lines purely for readbility.
-            // General sizes:
             "[--booker-main-width:480px] [--booker-timeslots-width:240px] lg:[--booker-timeslots-width:280px]",
-            "bg-muted grid max-w-full auto-rows-max items-start overflow-clip dark:[color-scheme:dark] md:flex-row",
-            layout === "small_calendar" &&
-              "w-[calc(var(--booker-meta-width)+var(--booker-main-width))] [--booker-meta-width:240px] lg:[--booker-meta-width:280px]",
-            // Sizes for fullscreen layouts:
+            layout === "small_calendar" && "[--booker-meta-width:240px] lg:[--booker-meta-width:280px]",
             layout !== "small_calendar" && "[--booker-meta-width:340px] lg:[--booker-meta-width:424px]",
             // Other styles
-            layout === "small_calendar" && "border-subtle min-h-[450px] rounded-md border",
-            layout !== "small_calendar" && "h-auto min-h-screen w-screen",
-            "transition-[width,max-width] duration-300"
+            "bg-muted grid max-w-full auto-rows-fr items-start overflow-clip dark:[color-scheme:dark] sm:transition-[width] sm:duration-300 sm:motion-reduce:transition-none md:flex-row",
+            layout === "small_calendar" && "border-subtle rounded-md border"
           )}>
           <AnimatePresence>
-            <StickyOnDesktop key="meta" className="relative z-10">
-              <BookerSection area="meta" className="max-w-screen w-full md:w-[var(--booker-meta-width)]">
+            <StickyOnDesktop key="meta" className="relative z-10 flex min-h-full">
```

Comment:
This line and the following lines help with pushing the calendar down in fullscreen view.